src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -126,6 +145,36 @@ public final void onRunFinish(boolean succeeded, BatchSinkContext context) {
     }
   }
 
+  private void recordMetric(BatchSinkContext context, Job queryJob) throws Exception {
+    String operation = baseConfiguration.get(BigQueryConstants.CONFIG_OPERATION);
+    String metricName = String.format("records.%s", operation.toLowerCase());
```

Comment:
It may also make sense to just have a single metric like 'records.updated' rather than one per operation 'records.insert', 'records.update', 'record.upsert'. That way people only ever look for the same metric rather than changing it per pipeline.

@sreevatsanraman @bdmogal do you have any opinions here?

Reply:
Updated the metric name to "records.updated"